draft-ietf-lamps-pkcs12-pbmac1.xml

```diff
@@ -221,6 +225,97 @@
+                 basic notation</title>
+          <author>
+              <organization>ITU-T
+              </organization>
+          </author>
+          <date month="February" year="2021"/>
+      </front>
+      <seriesInfo name="ITU-T Recommendation" value="X.680"/>
+      <seriesInfo name="ISO/IEC" value="8824-1:2021"/>
+      </reference>
+
+      <reference anchor="X681"
+                 target="https://www.itu.int/rec/T-REC-X.681">
+      <front>
+          <title>Information Technology - Abstract Syntax
+                 Notation One (ASN.1): Information object
+                 specification</title>
+          <author>
+              <organization>ITU-T
+              </organization>
+          </author>
+          <date month="February" year="2021"/>
+      </front>
+      <seriesInfo name="ITU-T Recommendation" value="X.681"/>
+      <seriesInfo name="ISO/IEC" value="8824-2:2021"/>
+      </reference>
+
+      <reference anchor="X682"
+                 target="https://www.itu.int/rec/T-REC-X.682">
+      <front>
+          <title>Information Technology - Abstract Syntax
+                 Notation One (ASN.1): Constraint specification</title>
+          <author>
+              <organization>ITU-T
+              </organization>
+          </author>
+          <date month="February" year="2021"/>
+      </front>
+      <seriesInfo name="ITU-T Recommendation" value="X.682"/>
+      <seriesInfo name="ISO/IEC" value="8824-3:2021"/>
+      </reference>
+
+      <reference anchor="X683"
+                 target="https://www.itu.int/rec/T-REC-X.683">
+      <front>
+          <title>Information Technology - Abstract Syntax
+                 Notation One (ASN.1): Parameterization of ASN.1
+                 specifications</title>
+          <author>
+              <organization>ITU-T
+              </organization>
+          </author>
+          <date month="February" year="2021"/>
+      </front>
+      <seriesInfo name="ITU-T Recommendation" value="X.683"/>
+      <seriesInfo name="ISO/IEC" value="8824-4:2021"/>
+      </reference>
+
+      <reference anchor="X690"
+                 target="https://www.itu.int/rec/T-REC-X.690">
+      <front>
+          <title>Information Technology - ASN.1 encoding rules:
+                 Specification of Basic Encoding Rules (BER),
+                 Canonical Encoding Rules (CER) and
+                 Distinguished Encoding Rules (DER)</title>
+          <author>
+              <organization>ITU-T
+              </organization>
+          </author>
+          <date month="February" year="2021"/>
+      </front>
+      <seriesInfo name="ITU-T Recommendation" value="X.690"/>
+      <seriesInfo name="ISO/IEC" value="8825-1:2021"/>
+      </reference>
+
+         <front>
+            <title>Information Technology - Abstract Syntax
+            Notation One (ASN.1):  Specification of basic notation.
+            ITU-T Recommendation X.680 (2002) | ISO/IEC 8824-1:2002. 

```

Comment:
This is a duplicated `<front>` section or missing opening `<reference>`. I can fix it, but not sure which title you want to use...

Reply:
A bad cut and paste error. This can be deleted.